py23-image/debian-10/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG SPLUNK_PRODUCT=splunk
+FROM ${SPLUNK_PRODUCT}-debian-10:latest
+USER root
+
+RUN apt update
+RUN apt-get install -y --no-install-recommends python3 python3-pip python3-setuptools python3-requests python3-yaml
```

Comment:
Just curious, which Python3 version does this give us? Are we going to try streamlining the same version across docker images with different operating system?

Reply:
It installs the latest version from each platform's default repo. It can varies on the minor version just like on what we have for python 2 installation currently.